packages/webdriver-utils/src/providers/automateProvider.js

```diff
@@ -146,21 +147,17 @@ export default class AutomateProvider extends GenericProvider {
     const tiles = [];
     const tileResponse = JSON.parse(responseValue.result);
     log.debug('Tiles captured successfully');
-    const windowHeight = responseValue?.metadata?.window_height || 0;
-    for (let tileData of tileResponse.sha) {
+    for (let tileData of tileResponse.tiles) {
       tiles.push(new Tile({
-        statusBarHeight: tileResponse.header_height || 0,
-        navBarHeight: tileResponse.footer_height || 0,
-        headerHeight,
-        footerHeight,
+        statusBarHeight: tileData.status_bar || 0,
+        navBarHeight: tileData.nav_bar || 0,
+        headerHeight: tileData.header_height || 0,
+        footerHeight: tileData.footer_height || 0,
         fullscreen,
-        sha: tileData.split('-')[0] // drop build id
+        sha: tileData.sha.split('-')[0] // drop build id
       }));
     }
-
-    const metadata = {
-      windowHeight: Math.ceil(windowHeight * dpr)
-    };
+    const metadata = {};
```

Comment:
are we adding anything to it later? can remove if not required

Reply:
We are adding cliExecutionTime in it later, which is used for performance metrics.